www/source/partials/docs/_glossary-supervisor.html.md.erb

```diff
@@ -18,3 +18,5 @@ An example is leader election: it's handled here by having a consistent (and sim
 ## Supervisor REST API
 
 The Habitat Supervisor provides a HTTP API to expose cluster metadata, statistics, and general diagnostic information useful for monitoring and support in the form of a JSON document. It also provides detailed information about the Habitat package that it is supervising, including metadata such as the build and runtime dependencies and their versions.
+
+<%= partial '/partials/docs/using-hab-monitor-services' -%>
```

Comment:
Another note on this. Because of the change that includes the glossary content into the header of the docs section (which I think is a pretty good idea) what we end up with is a very strange inclusion of monitoring the supervisor REST API at the very top of a docs page that i structured more along the lines of a chapter of a book. It definitely doesn't feel like the right thing looking at it in acceptance.

Reply:
That seems problematic, if we have to have multiple partials describing the exact same functionality because of a glossary or non-glossary location.

Reply:
Sorry I did a poor job of explaining this one. I definitely don't think we need multiple partials describing the same thing, I more meant that where those partials are now included when you look at this in acceptance feels bad. I wonder if we might need a refactor of the organization around a couple of things to get this corrected?